.licenserc.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+header:
+  # License format for Helm templates
+  - comment: on-failure
+    license:
+      spdx-id: Apache-2.0
+      copyright-owner: VMware, Inc.
+
+      content: |
+        {{- /*
+        Copyright VMware, Inc.
+        SPDX-License-Identifier: APACHE-2.0
+        */}}
+
+    paths:
+      - "bitnami/**/templates/**/*.yml"
+      - "bitnami/**/templates/**/*.yaml"
```

Comment:
Are we contemplating files directly in `/templates` with that approach or just files under another subdirectory `templates/foo`?
I.e `bitnami/postgresql/templates/read/svc.yaml` and `bitnami/postgresql/templates/svc.yaml`

Reply:
👍 Thanks